smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/HttpUriConflictValidator.java

```diff
@@ -159,6 +197,16 @@ private boolean isAllowableConflict(Model model, OperationShape operation, Opera
                 .collect(Collectors.toMap(Pair::getLeft, Pair::getRight));
     }
 
+    private Map<String, Pattern> getHostLabelPatterns(Model model, OperationShape operation) {
+        return OperationIndex.of(model).getInput(operation)
```

Comment:
Personal nit: I'd replace this with good old-fashioned looking Java. Maybe it's because the indentation implies that the Optional composition doesn't wrap the streaming composition.